docs/.eleventy.js

```diff
@@ -1,96 +1,96 @@
-const syntaxHighlight = require("@11ty/eleventy-plugin-syntaxhighlight")
+const syntaxHighlight = require("@11ty/eleventy-plugin-syntaxhighlight");
 
 const {
   EleventyHtmlBasePlugin,
   EleventyRenderPlugin,
-} = require("@11ty/eleventy")
-
-const yaml = require("js-yaml")
-const markdownIt = require("markdown-it")
-const markdownItEmoji = require("markdown-it-emoji")
-const markdownItAnchor = require("markdown-it-anchor")
-const pluginTOC = require("eleventy-plugin-toc")
-const now = String(Date.now())
-const path = require("path")
-const fs = require("fs")
-const mermaid = require("./_src/_plugins/mermaid")
-const nav = require("./_data/nav")
+} = require("@11ty/eleventy");
+
+const yaml = require("js-yaml");
```

Comment:
Could we do the other way around please? I'd rather not have to write the `;` if possible